### ISSUE TITLE
Extract subsets using filters (not just displaying them)?

### ISSUE THREAD

**Issue by alixdamman** [larray-project/larray-editor#37]
It would nice to have an "Extract" button next to the filters allowing to create a new variable to store the displayed subset.

**Comment by gdementen** [comment]
Yes, this seems like a great idea. I just wonder how it would interact with the (potential) move of the filters to the table headers? Where would the button go?

**Comment by alixdamman** [comment]
Why not via the context menu instead of a button?

**Comment by gdementen** [comment]
Good idea, but that will "call for" supporting extracting a selection, which would be great to have but we will only be able to do that once we support "sparse"/MultiIndex. Anyway, starting with a button would be fine.

**Comment by gdementen** [comment]
Please do not start working on this before discussing it, as it will be obsolete in the "new UI" I am designing.

**Comment by gdementen** [comment]
should be done after #73

**Comment by gdementen** [comment]
This is part of the "quickbar" feature.

**Comment by alixdamman** [comment]
I know but the "quickbar" is something big and will take time to develop.
My idea is to at least add this "Extract" button as soon as possible and then include it in the quickbar.

**Comment by gdementen** [comment]
Indeed, but the thing is the way to do it will be completely different, so we will basically develop the feature twice. Could be worth it, though. Has any user ever asked for it? If yes, then it is probably worth it to have it sooner than later.

**Comment by gdementen** [comment]
**IF** you think it is worth implementing twice, feel free to reassign yourself and move it back to the milestone you want it to be.

**Comment by gdementen** [comment]
For @yvdabb: the "quickbar" thing is very ambitious project (see #73) and probably not going to be implemented anytime soon, so feel free to implement this issue if you have time
